src/dbus/rauc.rs

```diff
@@ -237,6 +238,13 @@ async fn channel_list_update_task(
     let mut previous: Option<Instant> = None;
     let mut polling_tasks: Vec<JoinHandle<_>> = Vec::new();
 
+    // The tacd is usually started before the network connection is up.
+    // This means the first update polling request will nearly always fail.
+    // We could decide when the network is up based on some heuristic or
+    // based on what NetworkManager tells us.
+    // Or we can just wait a bit after startup.
+    sleep(STARTUP_DELAY).await;
```

Comment:
I'm not a fan of this hardcoded waiting, can't we decide on the error returned from `reload_stream` or another mechanism?

Reply:
I've replaced the hardcoded two minutes at startup by an exponential backoff mechanism that happens for each update poll. What do you think about that?